test/leader/leader_test.py

```diff
@@ -0,0 +1,74 @@
+from os import path
+
+import sys
+import unittest
+
+FILE_PATH = path.dirname(path.realpath(__file__))
+sys.path.append(FILE_PATH + "/../../src")
+
+from leader import Leader
+
+class LeaderTest(unittest.TestCase):
+    def test_set_speed_lower_bound(self):
+        leader = Leader()
+
+        self.assertEqual(leader.set_speed(-1), 0)
+
+
+    def test_set_speed_upper_bound(self):
+        leader = Leader()
+
```

Comment:
this blank space is unnecessary and makes the tests harder to follow.

Reply:
Should I remove all the blank spaces before the asserts? I followed Wisam's style, should I not?

Reply:
I removed the blank spaces, does that look better?